sormas-ui/src/main/java/de/symeda/sormas/ui/visit/VisitEditForm.java

```diff
@@ -88,12 +103,12 @@ public VisitEditForm(Disease disease, ContactDto contact, CaseDataDto caze, Pers
 
 	}
 
-	public VisitEditForm(Disease disease, ContactDto contact, PersonDto person, boolean create, boolean isInJurisdiction) {
-		this(disease, contact, null, person, create, contact.isPseudonymized());
+	public VisitEditForm(Disease disease, ContactDto contact, PersonDto person, boolean create) {
+		this(disease, contact, null, person, create, contact.isPseudonymized(), contact.isInJurisdiction());
```

Comment:
In the old version of `VisitController.java` there was a negation. Double check this.

Reply:
Yes, that's fine, the previous logic basically assumed that whenever an entry is not pseudonymized, it's automatically in the user's jurisdiction, which is technically wrong.